### PR TITLE
Merging branch with master

### PR DESCRIPTION
**Pull request by Evolution0**
As there currently is parity between the Python 3 fork and Python 2 branch I figured it would be faster as well as retain commit history if I simply merge the branch in then update the readme.

Just checking any last minute things before merging.

### CODE REVIEW

**Comment by Evolution0** [comment]
After this could you add me as a maintainer on PyPI so I can build and push packages?
1. View the package page, click roles.
2. Username: `AnthonyForsberg`
3. Role: Maintainer
4. Add role.

**Comment by iheanyi** [comment]
@Evolution0 Will do. Thanks for this.

File: bandcamp_dl/bandcamp_dl.py [hunk 1]
```diff
@@ -33,21 +31,26 @@
...
+    Shout out to darkf for writing a helper function for parsing the JavaScript!
+"""
 
-from docopt import docopt
-from Bandcamp import Bandcamp
-from BandcampDownloader import BandcampDownloader
 import os
+from docopt import docopt
+from Bandcamp_Parse import Bandcamp
+from Bandcamp_Download import BandcampDownloader
```
Comment (iheanyi) [inline bandcamp_dl/bandcamp_dl.py]: Any explicit reason why we renamed these modules? I think the names were fine as-is.

Comment (Evolution0) [reply bandcamp_dl/bandcamp_dl.py]: Mainly for clarity.

`from Bandcamp import Bandcamp` you have to admit tells you nothing when reading the code, now at first glance you know that you are importing a class related to parsing data from bandcamp.

As for Bandcamp_Download that was mainly for consistency and the fact that importing classes from modules of the same exact name seems a little odd. Though I later learned this is common (though with lower cased files, upper cased classes)

If you would like it changed back that is cool but I would recommend using the lowercase filename/uppercase class scheme.

Comment (iheanyi) [reply bandcamp_dl/bandcamp_dl.py]: Could we change it back to what they were before? I'm in agreement that it should be lowercased filenames + uppercase class names. :)

**Comment by Evolution0** [comment]
Refactored, original naming scheme restored (With lowercase filenames)

Should be ready to go, maybe give it a test and tell me how it goes? I've done testing myself but a second source of information is good.

I forgot to mention, you might notice that the IRC channel was removed and in its place is a link to Discord, its simply easier for people to connect to if they need help and the built-in voice channels is nice. If you hop on send me a DM and I'll add you as an admin.

**Comment by iheanyi** [comment]
@Evolution0 - Sorry, didn't have time to look this weekend. >_> <_< Shoot me an email with a link to it. :) @kwuchu on Twitter or you know my email through the email I sent ya

**Comment by Evolution0** [comment]
I tagged the release so the current working code is under releases, I even put together a wheel (though PyPI has issues with it)

Alternatively since I already released 0.0.5 on PyPI you can just pip install it now.

**Pull request merged**